src/subcommand/server.rs

```diff
@@ -451,7 +451,7 @@ impl Server {
           "this server has no rune index".to_string(),
         ));
       }
-      let runes = index.get_rune_balances_for_outpoint(outpoint)?;
+      let runes = index.get_rune_balances_for_output(outpoint)?;
```

Comment:
Any reason for the name update?

Reply:
No specific reason, just refactor to match the rest of the methods naming:

- PR: https://github.com/ordinals/ord/pull/3757/files#diff-701633ed548410624cfcae90d38a986adbd60deb5aeedd92355fe2e7702a9c18L2225-L2226